docs/product-installation-guide/src/main/asciidoc/install-standalone-properties-con.adoc

```diff
@@ -17,21 +24,17 @@
-* `org.uberfire.cluster.id`: Name of the helix cluster, for example: kie-cluster
-* `org.uberfire.cluster.zk`: Connection string to zookeeper. This is of the form host1:port1,host2:port2,host3:port3, for example: localhost:2188
-* `org.uberfire.cluster.local.id`: Unique id of the helix cluster node, note that ‘`:`’ is replaced with ‘`\_`’, for example: node1_12345
-* `org.uberfire.cluster.vfs.lock`: Name of the resource defined on helix cluster, for example: kie-vfs
-* `org.uberfire.cluster.autostart`: Delays VFS clustering until the application is fully initialized to avoid conflicts when all cluster members create local clones. Default: false
 * `org.uberfire.ldap.regex.role_mapper`: Regex pattern used to map LDAP principal names to application role name. Note that the variable role must be part of the pattern as it is substited by the application role name when matching a principal value to role name. Default: Not used.
 * `org.uberfire.sys.repo.monitor.disabled`: Disable configuration monitor (do not disable unless you know what you’re doing). Default: false
 * `org.uberfire.secure.key`: Secret password used by password encryption. Default`: org.uberfire.admin
 * `org.uberfire.secure.alg`: Crypto algorithm used by password encryption. Default: PBEWithMD5AndDES
 * `org.uberfire.domain`: security-domain name used by uberfire. Default: ApplicationRealm
 * `org.guvnor.m2repo.dir`: Place where Maven repository folder will be stored. Default: working-directory/repositories/kie
 * `org.guvnor.project.gav.check.disabled`: Disable GAV checks. Default: false
-* `org.kie.example.repositories`: Folder from where demo repositories will be cloned. The demo repositories need to have been obtained and placed in this folder. Demo repositories can be obtained from the kie-wb-6.2.0-SNAPSHOT-example-repositories.zip artifact. This System Property takes precedence over org.kie.demo and org.kie.example. Default: Not used.
-* `org.kie.demo`: Enables external clone of a demo application from GitHub. This System Property takes precedence over org.kie.example. Default: true
-* `org.kie.example`: Enables example structure composed by Repository, Organization Unit and Project. Default: false
+* `org.kie.example`: Enables external clone of a demo application from GitHub.
+* `org.kie.demo`: Enables external clone of a demo application from GitHub.
 * `org.kie.build.disable-project-explorer`: Disable automatic build of selected Project in Project Explorer. Default: false
 * `org.kie.verification.disable-dtable-realtime-verification`: Disables the realtime validation and verification of decision tables. Default: false
-* `org.kie.workbench.controller`: URL for connecting with a Kie Server Controller, for example: `ws://localhost:8080/kie-server-controller/websocket/controller`.
+* `org.kie.server.controller`: URL for connecting with a Kie Server Controller, for example: `org.kie.server.controller` - `ws://localhost:8080/decision-central/websocket/controller`.
```

Comment:
Please remove the name of the property from the example, it looks kind of confusing:
```
for example: `org.kie.server.controller` - `ws://localhost:8080/decision-central/websocket/controller`.

-> 
for example:  `ws://localhost:8080/decision-central/websocket/controller`.
```